### PR TITLE
prepare 1.38.0

### PR DESCRIPTION
**Pull request by r10s**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Delta Chat Android Changelog
 
+## v1.38.0
+2023-06
+
+* improve group membership consistency
+* fix verification issues because of email addresses compared case-sensitive sometimes
+* fix empty lines in HTML view
+* fix empty links in HTML view
```
Comment (adbenitez) [inline CHANGELOG.md:9]: duplicated line? :thinking:

File: src/org/thoughtcrime/securesms/ConversationListActivity.java [hunk 2]
```diff
@@ -101,12 +101,12 @@ protected void onCreate(Bundle icicle, boolean ready) {
     // it is not needed to keep all past update messages, however, when deleted, also the strings should be deleted.
     DcContext dcContext = DcHelper.getContext(this);
     DcMsg msg = new DcMsg(dcContext, DcMsg.DC_MSG_TEXT);
-    msg.setText("1.37 is out:\n\n"
+    msg.setText("1.38 is out:\n\n"
       + "📎 Removed upper size limit of attachments\n\n"
       + "🗜️ PNG images (screenshots!) are now compressed as well\n\n"
       + "⌨️ For Devs: Better apps for chats with sendToChat() and importFiles() (What is Webxdc? -> https://webxdc.org)\n\n"
       + "... and more at https://get.delta.chat/#changelogs");
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/ConversationListActivity.java:108]: shouldn't this message go to the strings.xml file for translation? :thinking:

**Review by adbenitez — APPROVED** [review]

File: src/org/thoughtcrime/securesms/ConversationListActivity.java (hunk 2 above)
Comment (r10s) [reply src/org/thoughtcrime/securesms/ConversationListActivity.java:108]: maybe, however, we do not always localize these messages ... and that one seems quite technical anyways ...

Comment (r10s) [reply src/org/thoughtcrime/securesms/ConversationListActivity.java]: but yes, before weighten for and against, i just added a commit and make it translatable, still makes sense

**Pull request merged**